pkg/controller/pod_driver.go

```diff
@@ -195,6 +196,10 @@ func (p *PodDriver) checkAnnotations(ctx context.Context, pod *corev1.Pod) error
 				klog.Errorf("Delete pod %s error: %v", pod.Name, err)
 				return err
 			}
+			// close socket
+			if util.ParseClientVersion(pod.Spec.Containers[0].Image).SupportFusePass() {
```

Comment:
can we extract this as a method?

`util.SupportFusePass(pod)`

Reply:
also, if the csi-driver does not mount the `jfs-fuse-fd` hostPath, can it be considered as not supported?

Reply:
The `jfs-fuse-fd` hostPath will be added in installation yaml and chart of new version by default. It can be ignored.

Reply:
worried about user only updated the image version